README.md

```diff
@@ -1,2 +1,24 @@
 # hw01
-Repository for first CFSS homework assignment
+  
+## James Marvel-Coen
+  
+### Brief Biography
+  
+#### Education and Affiliation
+* B.A., Williams College, 2011-2015
+* University of Chicago, 2015-**present**
+    * I am a second year PhD student in the [Department of Comparative Human Development.](http://humdev.uchicago.edu/)
+  
+#### Research Interests
+*My research interests include human behavior, cognition, and physiology in the context of evolutionary and developmental processes, and particularly the evolution and development of the human stress response.*
+
+#### How I made this.
+These are the steps I took to make this file:
+
+1. First, I forked the hw01 repo.
+
+2. Then, I created a new project in RStudio > Version Control > Git, and used the URL of the forked hw01 github repository as the repository URL.
+
+3. Then I wrote a brief biography of myself as instructed, and began describing how I did everything.
+
+4. After this, I will commit my changes, push back to github, and then submit a pull request on the original hw01 repo.
```

Comment:
I like your comment on how you did this step by step! The description of steps has proved that you have mastered this workflow!